src/Blazored.Modal/BlazoredModalInstance.razor.cs

```diff
@@ -275,12 +285,21 @@ private string SetModalClass()
     }
 
     private ModalAnimationType SetAnimation() 
-        => Options.AnimationType ?? GlobalModalOptions.AnimationType ?? ModalAnimationType.FadeInOut;
+        => Options.AnimationType ?? GlobalModalOptions.AnimationType ?? (!UseCustomLayout ? ModalAnimationType.FadeInOut : ModalAnimationType.None);
+
+    private string SetOverlayAnimationClass() => AnimationType switch
+    {
+        ModalAnimationType.FadeInOut or ModalAnimationType.FadeIn or
+        ModalAnimationType.MoveInOut or ModalAnimationType.MoveIn or
+        ModalAnimationType.PopInOut or ModalAnimationType.PopIn => "blazored-fade-in",
+        _ => string.Empty
+    };
 
-    private string SetAnimationClass() => AnimationType switch
+    private string SetModalAnimationClass() => AnimationType switch
     {
-        ModalAnimationType.FadeInOut => "fade-in",
-        ModalAnimationType.PopInOut or ModalAnimationType.PopIn => "pop-in",
+        ModalAnimationType.FadeInOut or ModalAnimationType.FadeIn => "blazored-fade-in",
+        ModalAnimationType.MoveInOut or ModalAnimationType.MoveIn => "blazored-move-in",
+        ModalAnimationType.PopInOut or ModalAnimationType.PopIn => "blazored-pop-in",
```

Comment:
can we revert the CSS class name changes, I can't imagine there is any value in changing them and it would make this a breaking change which I'd like to avoid

Reply:
Fixed